src/definitions/elements/input.less

```diff
@@ -455,20 +455,34 @@
   margin: 0;
 }
 
-/* Button on Right */
-.ui.action.input:not([class*="left action"]) > input {
+/* Input when ui Left*/
+.ui[class*="left"].action.input > input {
```

Comment:
Please change to
```css
.ui[class*="left action"].input > input {
```
Because it won't work with `<div class="ui right action input left icon">` otherwise